identifiers/country-mx.csv

```diff
@@ -0,0 +1,334 @@
+ocd-division/country:mx/state:ch,Chihuahua
+ocd-division/country:mx/state:ch/fed:1,I Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:2,II Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:3,III Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:4,IV Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:5,V Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:6,VI Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:7,VII Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:8,VIII Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:9,IX Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:cl,Colima
+ocd-division/country:mx/state:cl/fed:1,I Federal Electoral District of Colima
+ocd-division/country:mx/state:cl/fed:2,II Federal Electoral District of Colima
+ocd-division/country:mx/state:cm,Campeche
+ocd-division/country:mx/state:cm/fed:1,I Federal Electoral District of Campeche
+ocd-division/country:mx/state:cm/fed:2,II Federal Electoral District of Campeche
+ocd-division/country:mx/state:co,Coahuila
+ocd-division/country:mx/state:co/fed:1,I Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:2,II Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:3,III Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:4,IV Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:5,V Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:6,VI Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:7,VII Federal Electoral District of Coahuila
+ocd-division/country:mx/state:cs,Chiapas
+ocd-division/country:mx/state:cs/fed:1,I Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:10,X Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:11,XI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:12,XII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:13,XIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:2,II Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:3,III Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:4,IV Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:5,V Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:6,VI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:7,VII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:8,VIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:9,IX Federal Electoral District of Chiapas
+ocd-division/country:mx/state:df,Mexico City

```

Comment:
Line 57-81 'Mexico City' is listed as 'Ciudad de México' on INE site, would it be better to match?

Reply:
See above comment.